src/linear_algebra/basic.lean

```diff
@@ -755,6 +755,10 @@ noncomputable def equiv_map_of_injective (f : M →ₗ[R] M₂) (i : injective f
   map_smul' := by { intros, simp, refl, },
   ..(equiv.set.image f p i) }
 
+lemma coe_equiv_map_of_injective_apply (f : M →ₗ[R] M₂) (i : injective f)
```

Comment:
Isn't this the same as the lemma generated by `simps` above?

Reply:
Oh, I see it's not: https://leanprover-community.github.io/mathlib_docs/linear_algebra/basic.html#submodule.equiv_map_of_injective_apply

Can you make this a `simp` lemma, and remove the `simps` above?

Reply:
Or perhaps better, try `@[simps {simp_rhs := tt}]` and see if it produces a nicer lemma.